app/src/controllers/userController.ts

```diff
@@ -0,0 +1,127 @@
+import mongoose from 'mongoose';
+import { UserSchema } from '../models/userModel';
+import { Request, Response } from 'express';
+
+const User = mongoose.model('User', UserSchema);
+
+/**
+ * CRUD-Controller for user models
+ */
+export class UserController {
+
+    /**
+     * lists all available users
+     * @param  {Request} req - express request object
+     * @param  {Response} res — express response object
+     * @return json object with statusCode and list of users
+     */
+    public async index(req: Request, res: Response) {
```

Comment:
I think we don't need an API method that can return all registered users.

Reply:
Same as delete method - I think for administration in long run we'll need it and if not I would like to keep it for reference reasons until we create the other controllers. Ok?

Reply:
For now it will be a vulnerability to get all available users, or delete some user. You need to check for the admin permissions or create separate admin controller for that reasons. 

In my mind, it would better to remove this code and create an issues for adding such methods. Then we can add them carefully.